docs/protocol/src/concepts/addresses_keys.md

```diff
@@ -62,5 +62,4 @@ addresses, allowing fine-grained control of delegation.
 
 This diagram shows only the user-visible parts of the key hierarchy.
 Internally, each of these keys has different components, described in detail in
-the [Addresses and Keys](../protocol/addresses_keys.md) section of the
-[Cryptographic Protocol](../protocol.md) chapter.
+the [Addresses and Keys](../protocol/addresses_keys.md) chapter.
```

Comment:
Addresses and Keys is now its own chapter, rather than being under the Cryptographic Protocol chapter.